packages/cli/create-strapi-starter/src/utils/child-process.ts

```diff
@@ -0,0 +1,50 @@
+import { execSync } from 'child_process';
+import execa from 'execa';
+import logger from './logger';
+import type { Options } from '../types';
+
+export function runInstall(path: string, { useYarn }: Options = {}) {
+  return execa(useYarn ? 'yarn' : 'npm', ['install'], {
+    cwd: path,
+    stdin: 'ignore',
+  });
+}
+
+/**
+ * @param {string} rootPath
+ * @param {Object} options
+ * @param {boolean} options.useYarn
+ */
+export function runApp(rootPath: string, { useYarn }: Options = {}) {
+  if (useYarn) {
+    return execa('yarn', ['develop'], {
+      stdio: 'inherit',
+      cwd: rootPath,
+    });
+  }
+
+  return execa('npm', ['run', 'develop'], {
+    stdio: 'inherit',
+    cwd: rootPath,
+  });
+}
+
+export async function initGit(rootPath: string) {
+  try {
+    await execa('git', ['init'], {
+      cwd: rootPath,
+    });
+  } catch (err) {
+    logger.warn('Could not initialize a git repository');
+  }
+
+  try {
+    await execa('git', ['add', '-A'], { cwd: rootPath });
+
+    execSync('git commit -m "Create Strapi starter project"', {
+      cwd: rootPath,
+    });
```

Comment:
should we create a first commit? some companies use their own standard on commit messages?

Reply:
I don't think we want to init git without setting up a 1st commit but user can amend it if they really want.